src/scripts/main.js

```diff
@@ -2,7 +2,17 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.getElementsByTagName('tbody')[0];
```

Comment:
```suggestion
const table = document.querySelector('.dashboard tbody');
```
looks more understandable